### ISSUE TITLE
Cover Images not fetching from Google

### ISSUE THREAD

**Issue by cwldev** [janeczku/calibre-web#1494]
Running version 0.6.8 based on the linuxserver/calibre-web docker image.

When I go "fetch metadata" and cover results seem to be from Google, theya ren't shown or will not download.  See here for a screenshot: https://i.imgur.com/VbSxqq7.png

Seems possible Google changed something?

**Comment by OzzieIsaacs** [comment]
For me it's working:
![grafik](https://user-images.githubusercontent.com/6438895/84576785-a8c00b80-adb7-11ea-8108-c482b50fb40d.png)
What's the developer console in the browser telling us?

**Comment by cwldev** [comment]
Huh, mixed content errors. I run calibre-web with https enabled. Browser is Chrome 83.0.4103.97 (Official Build) (64-bit) on Windows 10.

Console: https://i.imgur.com/tbCJQ4I.png

Feels  like "site settings" should be the path to avoiding the error, but probably fetching the content by https is the best way: https://i.imgur.com/xiLR4Nh.png

**Comment by OzzieIsaacs** [comment]
It's google fault. The metadata request is done via https, in the response the links to the book are pointing to http addresses

**Comment by cwldev** [comment]
Indeed, I think that's the case. I continue to test and **most** cover images come through fine as long as I allow for non-encrypted content. It's all good.

**Comment by OzzieIsaacs** [comment]
Currently the pictures are fetched by the javascript code. On my secret list of  proposed changes is to do this by the server backend, so the mixed content could be prevented. Currently the number of issues is increasing, and I don't have enough time to fix everything.

**Comment by cwldev** [comment]
Sorry hear that... if I can help somehow, I will. Maybe I could take something off your list? Is there a list of stuff you're working on?

**Comment by flowck** [comment]
I am having the same issues. So far can't save any book cover.

![Screenshot 2020-11-28 at 00 22 29](https://user-images.githubusercontent.com/1679333/100489994-d6073880-310f-11eb-9946-0fd0e3e0e17e.png)
